index.rst

```diff
@@ -174,14 +174,14 @@
             <div class="col-sm-4 front-block-text">
                <h2>News</h2>
 
-               <ul>
+               <ul>            
                <li>
-                  2021-09-15:
-                  GMT now has an Instagram handle <a href="https://www.instagram.com/genericmappingtools/">@genericmappingtools</a>!
+                  2021-10-30:
```

Comment:
Maybe better to stick with the release date on the changelog.
```suggestion
                  2021-10-29:
```

Reply:
Good point! Thanks @weiji14 😉